### PR TITLE
(FACT-3165) Report correct location of where a fact got resolved from

### PR DESCRIPTION
**Pull request by chutzimir**

### CODE REVIEW

File: lib/facter/custom_facts/util/fact.rb [hunk 1]
```diff
@@ -223,8 +223,7 @@ def find_first_real_value(resolutions)
       end
 
       def log_fact_path(resolve)
-        fact = resolve.fact
-        log.debug("#{resolve.fact_type} fact #{fact.name} got resolved from: #{fact.location}")
+        log.debug("#{resolve.fact_type} fact #{resolve.fact.name} got resolved from: #{resolve.last_evaluated}")
```
Comment (joshcooper) [inline lib/facter/custom_facts/util/fact.rb:226]:
```suggestion
        log.debug("#{resolve.fact_type} fact #{resolve.fact.name} was resolved from: #{resolve.last_evaluated}")
```

**Comment by joshcooper** [comment]
Thank you for your contribution @chutzimir! Could you rebase your changes onto the head of the `main` branch which should resolve the CI failures

**Comment by chutzimir** [comment]
> Thank you for your contribution @chutzimir! Could you rebase your changes onto the head of the `main` branch which should resolve the CI failures

Rebased!

**Comment by chutzimir** [comment]
> Thank you for your contribution @chutzimir! Could you rebase your changes onto the head of the `main` branch which should resolve the CI failures

@joshcooper I rebased again, please let me know if anything else is needed.

**Comment by joshcooper** [comment]
retriggering tests

**Pull request merged**